Description: Fix spelling mistake
 s/emty/empty/g src/libcalamares/kdsingleapplicationguard/kdsingleapplicationguard.cpp
 This has been accepted upstream and will be in calamares 2.4.6.
 .
 calamares (2.4.5-1) unstable; urgency=low
Author: Jonathan Carter <jcarter@linux.com>
Bug-Debian: https://bugs.debian.org/842671

--- calamares-2.4.5.orig/src/libcalamares/kdsingleapplicationguard/kdsingleapplicationguard.cpp
+++ calamares-2.4.5/src/libcalamares/kdsingleapplicationguard/kdsingleapplicationguard.cpp
@@ -723,7 +723,7 @@ void KDSingleApplicationGuard::Private::
 
     const QString name = QCoreApplication::applicationName();
     if ( name.isEmpty() ) {
-        qWarning( "KDSingleApplicationGuard: QCoreApplication::applicationName must not be emty" );
+        qWarning( "KDSingleApplicationGuard: QCoreApplication::applicationName must not be empty" );
         return;
     }
 
